docs/overview.adoc

```diff
@@ -328,10 +328,10 @@ Adding on config the parameter `failOnError:false` (by default `true`), in case
 
 [cols="3m,2"]
 |===
-| CALL apoc.mongodb.get(host-or-port,db-or-null,collection-or-null,query-or-null,[compatibleValues=true|false]) yield value | perform a find operation on mongodb collection
+| CALL apoc.mongodb.get(host-or-port,db-or-null,collection-or-null,query-or-null,[compatibleValues=true\|false]) yield value | perform a find operation on mongodb collection
```

Comment:
@stdob would it make sense to add it here too?

Reply:
@jexp Yes, without escaping it broke the layout.

Reply:
You can also just use `/` instead :)

Reply:
I meant skip/limit to the `get` procedure.